xrpl/models/transactions/signer_list_set.py

```diff
@@ -97,11 +97,19 @@ def _get_errors(self: SignerListSet) -> Dict[str, str]:
         if self.signer_entries is None:  # deletion of the SignerList object
             return errors
 
-        if self.signer_quorum <= 0:
+        if self.signer_quorum == REQUIRED:
+            errors["signer_quorum"] = "`signer_quorum` is not set."
+        elif self.signer_quorum <= 0:
             errors[
                 "signer_quorum"
             ] = "`signer_quorum` must be greater than or equal to 0."
```

Comment:
This error says that `signer_quorum` must be greater than or equal to 0, but if it is 0 this error still happens. Is that a typo?

Reply:
Hm, not sure - it appears to have been the same previously.

Reply:
Looking at the code again, I think this condition is fine - line 97 returns `errors` if `signer_entries` is `None`, which should handle that situation.

Reply:
@mvadari I think updating the error to what @pdp2121 suggested makes sense here - 

Relevant quote from xrpl.org on SignerlistSet fields: (https://xrpl.org/signerlistset.html#signerlistset-fields) 
"You cannot create a signer list such that the SignerQuorum could never be met. The SignerQuorum must be greater than 0 but less than or equal to the sum of the SignerWeight values in the list. Otherwise, the transaction fails with the error [temMALFORMED](https://xrpl.org/tem-codes.html)."

Reply:
Error message was fixed in this commit: `e3b0f0eb9f262ed3ae3cdcbc5f6c59fc47432f3c`